Some warning fixes to EHCI code

There were a some ugly warnings in compiling this code.
Fixed those.

Signed-off-by: Remy Bohmer <linux@bohmer.net>
---
 drivers/usb/host/ehci-hub.c |    3 +++
 include/asm/arch/fsl_usb.h  |    2 +-
 2 files changed, 4 insertions(+), 1 deletion(-)

Index: linux-2.6.26/drivers/usb/host/ehci-hub.c
===================================================================
--- linux-2.6.26.orig/drivers/usb/host/ehci-hub.c	2009-03-15 12:14:31.000000000 +0100
+++ linux-2.6.26/drivers/usb/host/ehci-hub.c	2009-03-15 12:17:00.000000000 +0100
@@ -899,6 +899,7 @@ error:
 	return retval;
 }
 
+#if 0
 static void ehci_relinquish_port(struct usb_hcd *hcd, int portnum)
 {
 	struct ehci_hcd		*ehci = hcd_to_ehci(hcd);
@@ -918,3 +919,5 @@ static int ehci_port_handed_over(struct 
 	reg = &ehci->regs->port_status[portnum - 1];
 	return ehci_readl(ehci, reg) & PORT_OWNER;
 }
+#endif
+
Index: linux-2.6.26/include/asm-arm/arch-mxc/fsl_usb.h
===================================================================
--- linux-2.6.26.orig/include/asm-arm/arch-mxc/fsl_usb.h	2009-03-15 11:49:56.000000000 +0100
+++ linux-2.6.26/include/asm-arm/arch-mxc/fsl_usb.h	2009-03-15 11:52:16.000000000 +0100
@@ -73,7 +73,7 @@ static inline void fsl_platform_set_ahb_
 	if (cpu_is_mx35_rev(CHIP_REV_2_0) >= 1) {
 		temp = readl(hcd->regs + FSL_SOC_USB_TXFILLTUNING);
 		/* Change TX FIFO threshold to be 0x20 */
-		writel((temp & (~(0x3f << 16)) | (0x20 << 16)),
+		writel(((temp & (~(0x3f << 16))) | (0x20 << 16)),
 			hcd->regs + FSL_SOC_USB_TXFILLTUNING);
 	}
 }
